### DISCUSSION TITLE
Capacitor 5 or Capacitor 6 check whether Android 15 is supported?

### DISCUSSION THREAD

**Discussion by ColorGlaze** [ionic-team/capacitor#7799] [Q&A]
Capacitor 5 or Capacitor 6 check whether Android 15 is supported?

**Comment by jcesarmobile** [comment]
What do you mean by "supported"?

Both versions should be able to run on Android 15 devices without problems.

If what you want is to use target/compileSDK 15, then it's recommended to wait for Capacitor 7, we have already released a few alpha versions.

**Reply by Tomjech16** [reply]
Hi!
We’ve received a warning in the Google Play Console indicating that our app is using APIs or parameters that will be deprecated in Android 15. Specifically, it mentions the following:

- android.view.Window.setDecorFitsSystemWindows
- androidx.core.view.WindowCompat$Api30Impl.setDecorFitsSystemWindows
- android.view.Window.setNavigationBarColor

And others related to StatusBar and NavigationBar adjustments.

These appear to originate from the following Capacitor plugins:

@capacitor/splash-screen
@capacitor/status-bar
A third-party @capacitor/navigation-bar plugin.
While the app currently works without issues, the warnings suggest potential incompatibilities with edge-to-edge display handling in Android 15.

Could you confirm if these plugins (especially @capacitor/splash-screen and @capacitor/status-bar) will address the deprecated API usage in the stable release of Capacitor 7? Should we already consider migrating to the alpha versions of Capacitor 7 to ensure compliance?

Additionally, do you recommend any specific changes on our end to mitigate the use of deprecated APIs in the meantime, or should we focus solely on awaiting the official Capacitor 7 release?
